datasearch.py

```diff
@@ -64,9 +71,9 @@ def getPossWords(cellgrid):
     #if cellgrid.wword.poss_words:
     #    createPossWords(cellgrid.wword.poss_words)
     
-    print(f"DEBUG\tPossible words for '{cellgrid.wword.word}':\n\t{cellgrid.wword.poss_words}\n")
+    logger.debug(f"Possible words for '{cellgrid.wword.word}':\n\t{cellgrid.wword.poss_words}\n")
     if len(cellgrid.wword.poss_words) > 50:
-        print(f"\tTotal possible words: {len(cellgrid.wword.poss_words)}\n")
+        logger.info(f"\tTotal possible words: {len(cellgrid.wword.poss_words)}\n")
```

Comment:
Notice, the `print("DEBUG\t...")` lines were replaced with `logger.debug("...")` but here I'm using `logger.info("...")`.